learn-css-grid-by-building-a-magazine/index05.css

```diff
@@ -0,0 +1,7 @@
+.magazine {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+  /* The repeat() function allows you to reduce repetition like above. It is used in the following format:
+  repeat(no of times, value to be repeated)
```

Comment:
`no` -> `number`

Reply:
I've resolved this locally, would be pushed up with next update